Function-operators.Rmd

```diff
@@ -29,7 +29,7 @@ Function operators are closely related to function factories; indeed they're jus
 
 Function operators are typically paired with functionals. If you're using a for-loop, there's rarely a reason to use a function operator, as it will make your code more complex for little gain.
 
-If you're familiar with Python, decorators are just another name for function operators.
+If you're familiar with Python, _decorators_ is just another name for function operators.
```

Comment:
Why italics here?

Reply:
It was a correction from the editor. I can revert.

Reply:
Yeah, it seems inconsistent because I don't think we use italics in many places